spec/controllers/participations_controller_spec.rb

```diff
@@ -59,7 +59,7 @@
   describe '#destroy' do
     let(:post) { FactoryGirl.create(:status_message) }
 
-    context 'on a post you partecipate to' do
+    context 'on a post you participate to' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.